cisa-advisory/operations.py

```diff
@@ -0,0 +1,228 @@
+import json
+import requests
+from bs4 import BeautifulSoup
+import datetime
+from dateutil.relativedelta import relativedelta
+from connectors.core.connector import get_logger, ConnectorError
+from .constant import *
+from integrations.crudhub import make_request
+
+logger = get_logger('cisa-advisory')
+
+class Advisory():
+
+    def __init__(self, url):
+         self.url = url
+
+    def get_ics_data(self):
+        try:
+            ics_advisory_url_by_year_links_list = []
+            output = []
+            response = requests.get(self.url, verify=False)
```

Comment:
Ankit, you are hard coding the verify ssl here, please take its value from the configuration parameter.

Reply:
User don't have to provide URL